README.md

```diff
@@ -1,4 +1,4 @@
-# IO Functions template
+# IO Functions Backoffice
 
 Template per l'utilizzo di Azure Functions (e Durable Functions) all'interno del
 progetto IO.
```

Comment:
We have to modify this description :)

Reply:
The `README` was updated here a5ee491 with basic info